soc/v2/intel/intel_x86/soc.yml

```diff
@@ -0,0 +1,2 @@
+family:
+- name: intel_x86
```

Comment:
This might be a bit too generic and everything would fall under it, i.e. it almost matches the architecture and not soc families. There should be a way to define SoCs without families if I am not mistaken, ISH for example could be declared as family. We also have legacy stuff (ia32) that would need to be dealt up with and potentially cleaned up/dropped.

Reply:
changed to have no 'family' or 'series' for these SoC